### PR TITLE
Bump version to 3.0.4-3.0.0

### PR DESCRIPTION
**Pull request by glowkey**
* Update DCGM to 3.0.4
* Update headers to 3.0.4

### CODE REVIEW

File: deployment/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: dcgm-exporter
 description: A Helm chart for DCGM exporter
-version: "2.6.11"
+version: "3.0.0"
 kubeVersion: ">= 1.13.0-0"
```
Comment (nikkon-dev) [inline deployment/Chart.yaml:5]: I think it's time to update this. No way we are still supporting 1.13 API

**Review by nikkon-dev — COMMENTED** [review]
LGTM, but we need to update supported k8s version

File: deployment/Chart.yaml (hunk 1 above)
Comment (glowkey) [reply deployment/Chart.yaml]: This is updated to 1.19.

**Review by nikkon-dev — APPROVED** [review]

**Comment by hassanbabaie** [comment]
@glowkey is there any documentation related to the DCGM 3.0.4 release? I can't see this referenced anywhere in the Nvidia docs?

**Comment by glowkey** [comment]
The release notes have the major changes: https://docs.nvidia.com/datacenter/dcgm/latest/release-notes/changelog.html

**Comment by hassanbabaie** [comment]
thanks!, I'm sure that page didn't have v3 a minute ago;

**Pull request merged**